tests/bootstrap.php

```diff
@@ -50,3 +50,16 @@
 // does not allow the sessionid to be set after stdout
 // has been written to.
 session_id('cli');
+
+// Load schema from a SQL dump file.
+//
+// If your plugin does not use database fixtures you can
+// safely delete this.
+//
+// If you want to support multiple databases, consider
+// using migrations to provide schema for your plugin,
+// and using \Migrations\TestSuite\Migrator to load schema.
```

Comment:
Would be nice to have a code e.g. for this too.

Reply:
I'm going to add it later this week. That will be the 'default' mode for applications as I think re-using migrations is a better dev experience than having to generate a dump file. I was blocked on the migrations changes not being merged.